pycontrails/core/flight.py

```diff
@@ -2165,12 +2181,22 @@ def segment_rocd(
         See output from :func:`segment_duration`.
     altitude_ft: npt.NDArray[np.float64]
         Altitude of each waypoint, [:math:`ft`]
+    air_temperature: None | npt.NDArray[np.float64]
+        Air temperature of each flight waypoint, [:math:`K`]
 
     Returns
     -------
     npt.NDArray[np.float64]
         Rate of climb and descent over segment, [:math:`ft min^{-1}`]
 
+    Notes
+    -----
+    The hydrostatic equation will be used to estimate the ROCD if `air_temperature` is provided.
+    This will improve the accuracy of the estimated ROCD with a temperature correction. The
+    estimated ROCD with the temperature correction are expected to differ by up to +-5% compared to
+    those without the correction. These differences are important when the ROCD estimates are used
+    as inputs to aircraft performance models.
```

Comment:
Do you have any references that could be cited as to where this correction came from?